nym-node/src/config/template.rs

```diff
@@ -221,6 +221,10 @@ upstream_exit_policy_url = '{{ exit_gateway.upstream_exit_policy_url }}'
 
 [exit_gateway.storage_paths]
 
+# Path to sqlite database containing all persistent data: messages for offline clients,
```

Comment:
Is this description correct? I thought we removed messages for offline clients?

Reply:
I'm not sure tbh. If you know offline clients are removed, I can modify the description too, I just modified the wireguard part